pkgs/by-name/fo/follow/package.nix

```diff
@@ -30,22 +30,25 @@ stdenv.mkDerivation rec {
 
   pnpmDeps = pnpm.fetchDeps {
     inherit pname version src;
-    hash = "sha256-e9Ui3oIS0wbDLgntF7PQZCll12yvWeKLISXVoK6BjuE=";
+    hash = "sha256-FzMjN0rIjYxexf6tix4qi3mnuPkadjKihhN0Pj5y2nU=";
   };
 
   env = {
     ELECTRON_SKIP_BINARY_DOWNLOAD = "1";
 
     # This environment variables inject the production Vite config at build time.
     # Copy from:
-    # 1. https://github.com/RSSNext/Follow/blob/0745ac07dd2a4a34e4251c034678ace15c302697/.github/workflows/build.yml#L18
+    # 1. https://github.com/RSSNext/Follow/blob/v0.2.0-beta.2/.github/workflows/build.yml#L18
     # 2. And logs in the corresponding GitHub Actions: https://github.com/RSSNext/Follow/actions/workflows/build.yml
     VITE_WEB_URL = "https://app.follow.is";
     VITE_API_URL = "https://api.follow.is";
     VITE_IMGPROXY_URL = "https://thumbor.follow.is";
     VITE_SENTRY_DSN = "https://e5bccf7428aa4e881ed5cb713fdff181@o4507542488023040.ingest.us.sentry.io/4507570439979008";
-    VITE_BUILD_TYPE = "production";
-    VITE_POSTHOG_KEY = "phc_EZGEvBt830JgBHTiwpHqJAEbWnbv63m5UpreojwEWNL";
+    VITE_OPENPANEL_CLIENT_ID = "0e477ab4-d92d-4d6e-b889-b09d86ab908e";
+    VITE_OPENPANEL_API_URL = "https://openpanel.follow.is/api";
+    VITE_FIREBASE_CONFIG = ''
+      {"apiKey":"AIzaSyDuM93019tp8VI7wsszJv8ChOs7b1EE5Hk","authDomain":"follow-428106.firebaseapp.com","projectId":"follow-428106","storageBucket":"follow-428106.appspot.com","messagingSenderId":"194977404578","appId"":"1:194977404578:web:1920bb0c9ea5e2373669fb","measurementId":"G-SJE57D4F14"}
+    '';
```

Comment:
```suggestion
    VITE_FIREBASE_CONFIG = ''{"apiKey":"AIzaSyDuM93019tp8VI7wsszJv8ChOs7b1EE5Hk","authDomain":"follow-428106.firebaseapp.com","projectId":"follow-428106","storageBucket":"follow-428106.appspot.com","messagingSenderId":"194977404578","appId"":"1:194977404578:web:1920bb0c9ea5e2373669fb","measurementId":"G-SJE57D4F14"}'';
```

Reply:
@FliegendeWurst PTAL. I have refactored this config by adopting `builtins.toJSON`.